examples/rust/components/dog-fetcher/local.wadm.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: dog-fetcher
   annotations:
     version: v0.0.1
-    description: 'HTTP hello world demo in Rust, showing use of the server and client providers'
+    description: "HTTP hello world demo in Rust, showing use of the server and client providers"
```

Comment:
Generally all of our examples use single quotes instead of double quotes for the description, would you mind flipping this back for the sake of consistency?